src/smt/theory_str_noodler/formula_preprocess.cpp

```diff
@@ -540,7 +549,7 @@ namespace smt::noodler {
                 std::set<VarNode> occurs_act = this->formula.get_var_occurr(side[i]);
 
                 // do not include length variables
-                if(this->len_variables.find(side[i]) != this->len_variables.end()) {
+                if(false && this->len_variables.find(side[i]) != this->len_variables.end()) {
```

Comment:
We should resolve this better than passing `false` to the if statement.

Reply:
Sure, I will change it at some point